src/applications/pact-act/tests/cypress/service-period-during-both/form-validation.cypress.spec.js

```diff
@@ -3,7 +3,7 @@ import { ROUTES } from '../../../constants';
 
 // Note: anything requiring a VA button click is tested here as unit tests cannot
 // target the shadow DOM
-describe('PACT Act', () => {
+xdescribe('PACT Act', () => {
```

Comment:
I may be jumping ahead (I haven't had a chance to go through Slack yet). Is this test going to be addressed with the new format?

Reply:
Yep! Checkbox ticket will be cut today per Laura, and once we move off the DS components we can use activate this test again.

Reply:
Neat. Thanks!

Reply:
Oh, and we should add something to that ticket to ensure the test gets re-activated, please @laflannery

Reply:
@Randi and @chriskim2311 This is [the ticket for the checkboxes](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15902). I would not be able to add what you need here in any useful way, would one of you be able to update this?